mobilecoind/src/service.rs

```diff
@@ -2609,7 +2609,7 @@ mod test {
         // Add a block with a non-MOB token ID.
         add_block_to_ledger(
             &mut ledger_db,
-            BlockVersion::THREE,
+            BLOCK_VERSION,
```

Comment:
This used to be adding a block version 3 after block version 4 was already in the chain. Ledger will error out in these instances, https://github.com/mobilecoinfoundation/mobilecoin/blob/90560096c7cb306758b2d22bd4c5eb9891d20b66/ledger/db/src/ledger_db.rs#L695